Investigation/java/automl/.gitignore

```diff
@@ -0,0 +1,22 @@
+HELP.md
```

Comment:
I am not sure whether we should move it outside of Investigation folder or not, what do you think?

Reply:
"IntelliJ IDEA" should be outside, others seem to be this module related (keeps leaving them here?)

Reply:
Sorry, I didn't say that clearly. I mean moving all the files in this commit into a Java microservices folder instead of the Investigation folder.